src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageLoggerExtensions.cs

```diff
@@ -41,6 +42,10 @@ static PageLoggerExtensions()
                3,
                "Request was short circuited at page filter '{PageFilter}'.");
 
+            _redirectToPageResultExecuting = LoggerMessage.Define<string>(
+                LogLevel.Information,
+                1,
```

Comment:
@pranavkm - please follow our established convention. If there's an issue with our IDs then we will deal with it holistically when/if that bug is triaged.